android/app/src/main/java/org/fossasia/openevent/viewmodels/DayScheduleFragmentViewModel.java

```diff
@@ -0,0 +1,57 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.dbutils.RealmDataRepository;
+import org.fossasia.openevent.utils.SortOrder;
+
+import io.realm.RealmResults;
+
+/**
+ * Created by samagra on 23/1/18.
+ */
+
+public class DayScheduleFragmentViewModel extends ViewModel {
+    private MutableLiveData<RealmResults<Session>> realmResults;
+    private String searchText = "";
+
+
```

Comment:
Remove this extra line.

Reply:
What is wrong with blank lines. Already, the documentation is poor. There are no Javadoc comments. In such cases, blank lines can actually point out changes in context.

Reply:
I never said adding blank lines is wrong. Adding two/three of them where you should at max. add one line is wrong in my opinion. A single blank line can point out changes in context.

>  Already, the documentation is poor. There are no Javadoc comments.

**Then please help us make it better.** 
Please create a new issue if you think it should be present and maintainers will surely respond.

Reply:
Only one blank line is standard according to any Java-style guideline @samagra14